Added note on pagerduty alert notification (#37759)
grafana/grafana: docs/sources/alerting/old-alerting/notifications.md

> **Note:** The `state` tag overrides the current alert state inside the `custom_details` payload.

> > **Note:** Grafana uses the `Events API V2` integration. This can be configured for each service.

Reviewer: ```suggestion
> **Note:** Grafana uses the `Events API V2` integration. You can configure it for each service.
```
